tests/cases/fourslash/codeFixUnusedLabel_all.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+
+////label1: while (1) {}
```

Comment:
What happens for outdented labels on preceding lines?
```
label:
    code
```
Does formatting restore the indentation?  Otherwise, it seems like you'd end up with `code` outdented.